client/settings/deposits/index.js

```diff
@@ -129,6 +131,7 @@ const CustomizeDepositSchedule = () => {
 						value={ depositScheduleWeeklyAnchor }
 						onChange={ setDepositScheduleWeeklyAnchor }
 						options={ daysOfWeek }
+						__nextHasNoMarginBottom
```

Comment:
`36px default size for wp.components.SelectControl is deprecated since version 6.8 and will be removed in version 7.1.`
```suggestion
					__nextHasNoMarginBottom
					__next40pxDefaultSize
```